### ISSUE TITLE
multiproc without specifying extension

### ISSUE THREAD

**Issue by dshean** [samapriya/porder#3]
Would be better to allow multiprocessing download of all available image, metadata, RPC, etc files for each id, rather than requiring the user to limit to one extension type.  Should be straightforward to implement.

**Comment by samapriya** [comment]
I have now allowed for the user to specify either all download or a specific extension on all downloaders now along with some general improvements.

**Comment by rbavery** [comment]
The extension option is required currently when using multiproc, whereas the docs state that this is optional

```
Traceback (most recent call last):
  File "/home/rave/anaconda3/envs/porder-env/bin/porder", line 11, in <module>
    sys.exit(main())
  File "/home/rave/anaconda3/envs/porder-env/lib/python2.7/site-packages/porder/porder.py", line 201, in main
    args.func(args)
  File "/home/rave/anaconda3/envs/porder-env/lib/python2.7/site-packages/porder/porder.py", line 110, in multiproc_from_parser
    subprocess.call("python multiproc_pydl.py "+args.url+" "+args.local+" "+args.ext,shell=True)
TypeError: cannot concatenate 'str' and 'NoneType' objects
```

**Comment by samapriya** [comment]
Thanks @rbavery I have now fixed in version 0.1.2
